_pytest/capture.py

```diff
@@ -308,6 +308,14 @@ class FDCapture:
     """ Capture IO to/from a given os-level filedescriptor. """
 
     def __init__(self, targetfd, tmpfile=None):
+        # ensure readline is imported so that it attaches to the correct
+        # stdio handles on Windows
+        if targetfd in (0, 1, 2):
```

Comment:
I'll add a link back to this when I report the issue.  But for now I'm heading home :)